### PR TITLE
Azure pipelines user profile api

### PR DESCRIPTION
**Pull request by zaol95**

### CODE REVIEW

File: azure-pipelines-userProfile.yml [hunk 1]
```diff
@@ -36,4 +41,5 @@ stages:
         dockerfile: $(dockerfilePath)
         containerRegistry: $(dockerRegistryServiceConnection)
         tags: |
-          $(tag)
+          $(Build.BuildId)
+          latest
```
Comment (miguel3231) [inline azure-pipelines-userProfile.yml]: remove latest tag

**Review by miguel3231 — APPROVED** [review]

File: azure-pipelines-userProfile.yml (hunk 1 above)
Comment (zaol95) [reply azure-pipelines-userProfile.yml]: removed from yml file

**Review by phillipstr — APPROVED** [review]
Approved

**Review by ninjaorlandolopez — APPROVED** [review]
ok

**Pull request merged**